src/status_im/android/platform.cljs

```diff
@@ -20,6 +20,10 @@
                            :modal       {:height       0
                                          :bar-style    "light-content"
                                          :color        styles/color-black}
+                           ;;TODO because this bug in RN https://github.com/facebook/react-native/issues/7474
+                           :modal-white {:height       0
+                                         :bar-style    "light-content"
+                                         :color        styles/color-black}
```

Comment:
Looks like color is wrong?

Reply:
why?

Reply:
if you mean it should be white, but it doesn't work on Android, and black looks back than default gray

Reply:
Hmm ok. BTW this issue is related https://github.com/status-im/status-react/issues/1829
Maybe you can fix it at the same time?

Reply:
sure